cvat/apps/dataset_manager/annotation.py

```diff
@@ -182,6 +183,10 @@ def reset(self):
         self.shapes = []
         self.tracks = []
 
+    @property
+    def is_stream(self) -> bool:
+        return not isinstance(self.shapes, list)
```

Comment:
It would probably be better to change the type annotation to `Iterable | list` in this case, otherwise I'd prefer using `inspect.isgenerator()`.